lib/app/widgets/player/player_widget.dart

```diff
@@ -67,15 +67,15 @@ class AudioInfoWidget extends StatelessWidget {
         stream: player.currentIndexStream,
         builder: (context, snapshot) {
           final index = snapshot.data ?? 0;
-          final track = player.tracks[index];
+          final track = (index >= 0 && index < player.tracks.length) ? player.tracks[index] : null;
 
           return Column(
             crossAxisAlignment: CrossAxisAlignment.start,
             children: [
               Text(
-                track.title,
+                track?.title ?? 'Unknown Title',
               ),
-              Text(track.artist),
+              Text(track?.artist ?? 'Unknown Artist'),
```

Comment:
Would you mind not use hardcoded msgs but instead add new entries for **artist**, **album** and **title** in `l10_en.arb` file, run `flutter gen-l10n` and then use them from `RetipL10n` mixin?